modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -3358,6 +3358,7 @@ choose-billing-address=Choose Billing Address
 choose-content=Choose Content
 choose-content-export-help=Only the content and metadata of the selected applications is exported. Use the <em>Change</em> option to disable the export of content and metadata for each application.
 choose-content-publish-help=Only the content and metadata of the selected applications is published. Use the <em>Change</em> option to disable the publication of content and metadata for each application.
+choose-for-how-long-in-minutes-the-result-of-the-script-will-be-cached=Choose for how long in minutes the result of the script will be cached
```

Comment:
@ricardocousolr Following BChan's best practices with language keys, every sentence needs to end with an ending ponctuation mark, so we need to add a `.` here.